src/components/Model/ModelDiscussion/ReviewDiscussionItem.tsx

```diff
@@ -46,6 +47,7 @@ export function ReviewDiscussionItem({ review }: Props) {
     { id: review.id },
     { initialData: review._count.comments }
   );
+  const { data: model } = trpc.model.getById.useQuery({ id: review.modelId });
```

Comment:
This one too. I like that it's this easy to get the data, I just am not sure if there are efficiency issues with doing this rather than passing things down through props or something...